WebContent/register.jsp

```diff
@@ -104,9 +189,27 @@
 					</div>
 				</div>
 			</form>
+			</div>
 		</div>
 	</div>
-
+	<script>
+		$(document).ready(function(){
+			$('#userType').change(function(){
+				var selectedType = $(this).val();
+				
+				if(selectedType == 'company'){
+					$('#companyFields').show();
+					$('#studentFields').hide();
+				} else if(selectedType == 'student'){
+					$('#companyFields').hide();
+					$('#studentFields').show();
+				}else {
```

Comment:
why do we have an else here? There should just be a default ``selectedType`` which would probably be student. 
I think you should edit this a bit.

Reply:
that makes sense ill remove the else, not necessary